data/1.0.0-preview-1/schema/lagrange_coefficients_record.schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "title": "ParsingModel[LagrangeCoefficientsRecord]",
+  "$ref": "#/definitions/LagrangeCoefficientsRecord",
+  "definitions": {
+    "ElementModQ": {
+      "title": "ElementModQ",
```

Comment:
This is actually wrong. Data is no longer included. Check the other files to ensure this mistake isn't repeated.
```
"ElementModQ": {
      "title": "ElementModQ"
       "type": "string"
       }
```

Reply:
Nice catch!  It actually wasn't part of the ticket, I just copied it over from the .95 spec.  There are no other files like that.